api/events.js

```diff
@@ -30,22 +32,28 @@ router.get("/:id/", async (req, res, next) => {
 });
 //AUTH update event:id (add attendees)
 
-router.post("/:id/attendingUsers", authenticate, async (req, res, next) => {
-  //post current user to attendingUser
+router.patch("/:id", authenticate, async (req, res, next) => {
   const { id } = req.params;
-  //const { userId } = req.user.id;
+  const { attending } = req.body;
+
   try {
     //const user = userId.map((id) => ({ id }));
     const event = await prisma.event.update({
       where: {
         id: +id,
       },
       data: {
-        attendingUsers: {
-          connect: {
-            id: req.user.id,
-          },
-        },
```

Comment:
logic for connecting the users id if attending is true and disconnecting if attending is false.